packages/contracts/src/framework/plugin/repo/PluginRepo.sol

```diff
@@ -272,4 +273,7 @@ contract PluginRepo is
             _interfaceId == type(IProtocolVersion).interfaceId ||
             super.supportsInterface(_interfaceId);
     }
+
+    /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting down storage in the inheritance chain (see [OpenZeppelin's guide about storage gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
+    uint256[46] private __gap;
```

Comment:
For everyone else for sensemaking:
```solidity
    mapping(uint8 => uint16) internal buildsPerRelease;
    mapping(bytes32 => Version) internal versions;
    mapping(address => bytes32) internal latestTagHashForPluginSetup;
    uint8 public latestRelease;
```
Mappings (and dynamic arrays) occupy 1 slot each (see https://docs.soliditylang.org/en/v0.8.25/internals/layout_in_storage.html#mappings-and-dynamic-arrays).
The `uint8` opens a new slot as well.